File: mobile-api-base/lib/core/config/api_config.dart
class ApiConfig {
  static const String baseUrl =
      String.fromEnvironment('MOTCHILL_PUBLIC_API_BASE_URL', defaultValue: 'https://motchilltv.taxi');
  static const Duration requestTimeout = Duration(seconds: 20);

  static Map<String, String> headers() {
    return const {
      'User-Agent': 'Mozilla/5.0 (MotchillApiBase)',
      'Accept': 'application/json,text/plain,*/*',
    };
  }
}

File: android-compose/app/src/main/java/com/motchill/androidcompose/core/config/ApiConfig.kt
package com.motchill.androidcompose.core.config

import com.motchill.androidcompose.BuildConfig
import java.util.concurrent.TimeUnit

object ApiConfig {
    val baseUrl: String = BuildConfig.MOTCHILL_PUBLIC_API_BASE_URL.trimEnd('/')
    val requestTimeoutMillis: Long = TimeUnit.SECONDS.toMillis(20)

    fun headers(): Map<String, String> {
        return mapOf(
            "User-Agent" to "Mozilla/5.0 (MotchillAndroidCompose)",
            "Accept" to "application/json,text/plain,*/*",
        )
    }
}

